src/components/Table/hoc/withTableSorting/SortIndicator/SortIndicator.scss

```diff
@@ -3,11 +3,7 @@
 $block: '.#{variables.$ns}sort-indicator';
 
 #{$block} {
-    &__caret {
-        padding: 1px 0;
-
-        & > svg {
-            display: block;
-        }
+    &__icon {
+        vertical-align: -2px;
```

Comment:
Thats very unstable solution. Maybe it will be more robust to set `sort-indicator` as flex and align icon inside it?

Reply:
But we don't want it to be centered, we actually want it to be aligned with baseline, -2px moves the icon slightly below the baseline like a normal letter